packages/ts-transform/src/utils/css-transform.tsx

```diff
@@ -0,0 +1,89 @@
+import postcss, { plugin } from 'postcss';
+import autoprefixer from 'autoprefixer';
+import cssnano from 'cssnano-preset-default';
+import nested from 'postcss-nested';
+import whitespace from 'postcss-normalize-whitespace';
+
+const minify = () => {
+  const preset = cssnano();
+  // We exclude async because we need this to run synchronously as ts transformers aren't async!
+  const asyncPluginsToExclude = ['postcss-svgo'];
+
+  return preset.plugins
+    .map(([creator]: any) => {
+      // replicate the `initializePlugin` behavior from https://github.com/cssnano/cssnano/blob/a566cc5/packages/cssnano/src/index.js#L8
+      return creator();
+    })
+    .filter((plugin: any) => {
+      return !asyncPluginsToExclude.includes(plugin.postcssPlugin);
+    });
+};
+
+const parentOrphenedPseudos = plugin('parent-orphened-pseudos', () => {
+  return root => {
+    root.walkRules(rule => {
+      if (rule.selector.includes(':')) {
+        const newSelector = rule.selector
+          .split(',')
+          .map(part => {
+            if (part.match(/^. /)) {
+              // If the selector has one characters with a space after it, e.g. "> :first-child" then return early.
+              return part;
+            }
+
+            if (part.includes(':')) {
```

Comment:
wouldnt it be easier and more performant to just check `part.charCodeAt(0) === 58` (58 is a char code for `:`)?

Reply:
you probably wouldnt need the previous if statements at all using this strategy

Reply:
cheers :)

hows it go vs. checking `part[0] === ':'`?

Reply:
it's equivalent - but `.charCodeAt` according to benchmarks is faster, why? no idea 😅maybe it's just because comparing numbers is more optimized than comparing strings? 🤷‍♂️